### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [git-ecosystem/git-credential-manager#1010] [Q&A] [answered]
Hi everyone.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by ldennington** [comment, marked as answer]
Yes, it is. Please feel free to [open issues](https://github.com/GitCredentialManager/git-credential-manager/issues/new/choose) or (even better) [submit PRs](https://github.com/GitCredentialManager/git-credential-manager/pulls) with proposed fixes and new features!

**Reply by ghost** [reply]
Hi ldennington.

> Yes, it is. Please feel free to [open issues](https://github.com/GitCredentialManager/git-credential-manager/issues/new/choose) or (even better) [submit PRs](https://github.com/GitCredentialManager/git-credential-manager/pulls) with proposed fixes and new features!

thank you for feedback.
